content/learn/book/the-game-loop/time-and-timers.md

```diff
@@ -0,0 +1,277 @@
++++
+title = "Time and Timers"
+insert_anchor_links = "right"
+[extra]
+weight = 5
++++
+
+Responding to the passage of time is essential for gameplay, animations and audio.
+But doing so correctly is surprisingly subtle.
+
+The [`Time`] resource in Bevy is the source of truth
+for the "current in-game time".
+Unlike [`Instant::now()`], the [`Time`] is fixed for the entire frame,
+rather than reflecting the precise wall clock time at the time of system evaluation.
+This value is set at the start of the frame based on the time supplied by the renderer,
+during the [`TimeSystem`] system set in the [`First`] schedule.
```

Comment:
Not sure how I'd reword this, but `Time` isn't exactly fixed for the entire frame, since it uses a different clock depending on the schedule, like is described later on. I understand what the intent here is though

---

Extra nit: This uses `TimeSystem`, which is the name in 0.16, even though it's `TimeSystems` on `main`. But later on the `RunFixedMainLoop` name is used, which is the new name on `main`. So there's some inconsistency in using 0.16 and `main` types